app/models/student.py

```diff
@@ -24,22 +30,13 @@ def __repr__(self):
         return f"{self.name}:{self.clas}-{self.division}"
     
     
-    
-studentFields={
-    'sid':fields.Integer,
-    'name':fields.String,
-    'clas':fields.Integer,
-    'division':fields.String
-}    
-
-
-# Students class
-class Students(Resource):
+    @staticmethod
     @marshal_with(studentFields)
-    def get():
+    def get_student():
         students=SMS.query.all()
         return students
     
+    @staticmethod
     @marshal_with(studentFields)
     def post():
```

Comment:
Why the name of the method is post() here? @siraj-shaikh-BS

Reply:
Okay i'll change it to add_student